### DISCUSSION TITLE
Some users receiving "this site can't provide a secure connection" errors

### DISCUSSION THREAD

**Discussion by mrispoli24** [vercel/community#5080] [Help] [answered]
### Page to Investigate

https://redpanda.com

### Steps to Reproduce

We're having some users receive the following error when attempting to access our site.

![image](https://github.com/vercel/community/assets/5890462/76fbe073-739c-4be6-939c-d56cd6d10973)

We're seeing this for a number of customers and seems to be related to this issue: https://www.reddit.com/r/nextjs/comments/1418l25/vercelcom_blacklisted_by_a_major_isp/

What's the best way to have this resolved?

**Comment by amyegan** [comment]
Hi @mrispoli24. Is the problem limited to specific regions or more common in one browser than in others? Sharing any patterns you have seen can help with investigation 🙏

**Comment by BetulBuyukzengin** [comment]
Me and my friends around me are also experiencing the same problem. I tried in all browsers but could not access it.
Region: Türkiye

**Comment by HasanMothaffar** [comment]
We're having a similar issue at work. Users in Turkey are receiving the error above, but users in other regions (including me) can access our app without any problem. A contact I know in Turkey changed their ISP and it worked, so it seems that this is an ISP related issue.

**Comment by swarnava** [comment, marked as answer]
Hi,

This looks like indeed ISP level block indeed and we are investigating currently. We are locking this thread and share further update here once we hear back from ISP: https://github.com/orgs/vercel/discussions/5102.

Thank you!

**Reply by erdalugur** [reply]
same

**Reply by swarnava** [reply]
Hi,

We are locking this thread and share further update here once we hear back from ISP: https://github.com/orgs/vercel/discussions/5102.

Thank you!
